meson.build

```diff
@@ -6,9 +6,16 @@ akira_datadir = join_paths(akira_prefix, get_option('datadir'))
 akira_pkgdatadir = join_paths(akira_datadir, meson.project_name())
 
 add_project_arguments(
-    ['--vapidir', join_paths(meson.current_source_dir(), 'vapi')],
     language: 'vala'
 )
+
+vala_version = meson.get_compiler('vala').version()
+if vala_version.version_compare('<=0.43.90')
+    add_project_arguments(
+        ['--vapidir', join_paths(meson.current_source_dir(), 'vapi')]
+    )
+endif
```

Comment:
I guess the `add_project_arguments` doesn't add extra arguments but overrides what was defined before, as the `language` argument is missing when the vala version is <= 0.43.90

Reply:
The language should be moved to the project definition at the very top of the meson.build file to avoid this.

Reply:
Ah wait, the language is already defined in the project definition at the beginning.
If I'm not wrong, the `language` attribute is only necessary if we're defining the `vapi`.
I'll give it a try.

Reply:
Yes, I was right.
Does it build normally with an updated version of Vala?

Reply:
It does.